examples/webpack.config.js

```diff
@@ -48,9 +48,11 @@ module.exports = (env, args) => {
         {
           test: /\.css$/,
           use: prod
-            ? ExtractTextPlugin.extract({
-                fallback: "style-loader",
-                use: {
```

Comment:
There are duplicated css-loaders. You can reduce to a single.

```js
use: [
  prod ? MiniCssExtractPlugin.loader : "style-loader",
    {
      loader: "css-loader",
      options: {
        sourceMap: !prod,
        // minimized by OptimizeCssAssetsPlugin
        minimize: false,
      },
    },
],
```

Reply:
Okay, I'll fix it!